.git-blame-ignore-revs

```diff
@@ -162,3 +162,6 @@ fc7a83f8b62e90de5679e993d4d49ca014ea013d
 
 # darwin.stdenv: format with nixfmt-rfc-style (#333962)
 93c10ac9e561c6594d3baaeaff2341907390d9b8
+
+# jnv: format with nixfmt-rfc-style (#334827)
```

Comment:
Not sure if we should append each package reformat here and not just mass-formattings.

Reply:
Not sure either, just going with what I've seen happening elsewhere.

Reply:
Sounds good to me, thanks!